apps/set-batch-overrides/src/set_batch_overrides.py

```diff
@@ -66,15 +81,16 @@ def lambda_handler(event: dict, _) -> dict:
     job_type, job_parameters = event['job_type'], event['job_parameters']
 
     if job_type == 'INSAR_ISCE_MULTI_BURST':
-        memory = get_insar_isce_burst_memory(job_parameters)
-        omp_num_threads = INSAR_ISCE_BURST_OMP_NUM_THREADS[memory]
-        return get_container_overrides(memory, omp_num_threads)
-
-    if job_type == 'AUTORIFT' and job_parameters['granules'][0].startswith('S2'):
-        return get_container_overrides(AUTORIFT_S2_MEMORY)
+        burst_memory = get_insar_isce_burst_memory(job_parameters)
+        omp_num_threads = INSAR_ISCE_BURST_OMP_NUM_THREADS[burst_memory]
+        return get_container_overrides(burst_memory, omp_num_threads)
 
-    if job_type == 'AUTORIFT' and job_parameters['granules'][0].startswith('L'):
-        return get_container_overrides(AUTORIFT_LANDSAT_MEMORY)
+    if job_type == 'AUTORIFT':
+        autorift_memory = get_autorift_memory(job_parameters)
+        if autorift_memory is not None:
+            # vCPU = Memory in GB / 8 for r6 instance types
+            omp_num_threads = str(ceil(int(autorift_memory) / 8000))
```

Comment:
This is the same logic for INSAR_ISCE_MULTI_BURSTS, or any job running on the `r` instance family.